contracts/core/liquidators/TWAPLiquidator.sol

```diff
@@ -0,0 +1,291 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+pragma experimental "ABIEncoderV2";
+
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import { BoundsLibrary } from "set-protocol-contract-utils/contracts/lib/BoundsLibrary.sol";
+
+import { ICore } from "../interfaces/ICore.sol";
+import { ILiquidator } from "../interfaces/ILiquidator.sol";
+import { IOracleWhiteList } from "../interfaces/IOracleWhiteList.sol";
+import { ISetToken } from "../interfaces/ISetToken.sol";
+import { Auction } from "./impl/Auction.sol";
+import { LinearAuction } from "./impl/LinearAuction.sol";
+import { Rebalance } from "../lib/Rebalance.sol";
+import { RebalancingLibrary } from "../lib/RebalancingLibrary.sol";
+import { AuctionGetters } from "./impl/AuctionGetters.sol";
+import { TWAPAuctionGetters } from "./twap-impl/TWAPAuctionGetters.sol";
+import { TWAPAuction } from "./twap-impl/TWAPAuction.sol";
+import { TwoAssetPriceBoundedLinearAuction } from "./impl/TwoAssetPriceBoundedLinearAuction.sol";
+
+
+/**
+ * @title TWAPLiquidator
+ * @author Set Protocol
+ *
+ * Contract that holds all the state and functionality required for setting up, returning prices, and tearing
+ * down TWAP rebalances for RebalancingSetTokens.
+ */
+contract TWAPLiquidator is
+    ILiquidator,
+    TWAPAuction,
+    AuctionGetters,
```

Comment:
Don't need this since TWAPAuctionGetters inherits it

Reply:
+1